tests/components/reconciler/Makefile

```diff
@@ -0,0 +1,12 @@
+ifdef DEBUG
+VERBOSE = -v
+endif
+
+.PHONY: resolve
+resolve:
+	go mod tidy -compat=1.17
```

Comment:
This is unnecessary, you can already base the module on go1.19 without doing this

Reply:
fixed :)